app/components/pages/ThankYou/ThankYou.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Box } from 'grid-styled'
+import PropTypes from 'prop-types'
+import { H1 } from '@pubsweet/ui'
+import { th } from '@pubsweet/ui-toolkit'
+import ExternalLink from '../../ui/atoms/ExternalLink'
+import ButtonLink from '../../ui/atoms/ButtonLink'
+import Paragraph from '../../ui/atoms/Paragraph'
+import SmallParagraph from '../../ui/atoms/SmallParagraph'
+
+const CenteredContent = styled(Box)`
+  text-align: center;
+`
+const SubText = styled(SmallParagraph)`
+  color: ${th('colorTextSecondary')};
+`
+
+const BookmarkLink = styled(ExternalLink)`
+  display: inline-block;
```

Comment:
Just an FYI, it looks like this on firefox
![image](https://user-images.githubusercontent.com/15656538/46955684-6ba7f100-d08b-11e8-83b9-ce0032e0f374.png)